recording_server/src/media_context_macos.ts

```
@@ -0,0 +1,294 @@
+    private process: ChildProcess | null
+    private promise: Promise<number> | null
+
+    constructor() {
+        this.process = null
+        this.promise = null
+    }
+
+    protected execute(
+        args: string[],
+        after: { (): void },
+    ): ChildProcess | null {
```

Biome (1.9.4)

[error] Use a function type instead of a call signature.

Types containing only a call signature can be shortened to a function type.
Safe fix: Use a function type instead of an object type with a call signature.


(lint/style/useShorthandFunctionType)